x-pack/plugins/endpoint/server/services/endpoint/endpoint_query_builders.ts

```diff
@@ -66,6 +68,15 @@ async function getPagingProperties(
   };
 }
 
+function buildQueryBody(request: KibanaRequest<any, any, any>): Record<string, any> {
+  if (request?.body?.filters && typeof request.body.filters === 'string') {
```

Comment:
I'm new to the ? operator - very cool. Could this be shortened to just `typeof request?.body?.filters === 'string'`?